.github/workflows/pr_index_integrity.yaml

```diff
@@ -0,0 +1,18 @@
+name: GH Pages Integrity
+
+on:
+  pull_request:
+    branches:
+      - gh-pages
+
+jobs:
+  gh-pages-integrity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: check index.yaml
+        run: |
+          helm repo add test https://raw.githubusercontent.com/newrelic/helm-charts/${{ github.event.pull_request.base.ref }}
+          helm search repo newrelic -l --devel
```

Comment:
Repo is added as `test` but tested as `newrelic`. Is this intended?

Reply:
is looking for all the images "newrelic"

```
NAME                             	CHART VERSION	APP VERSION	DESCRIPTION                                       
test/newrelic-infra-operator     	0.4.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.3.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.2.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.3        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.2        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.1        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.0        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.8.0        	2.9.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.3        	2.8.3      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.2        	2.8.2      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.1        	2.8.2      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.0        	2.8.1      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.4        	2.7.1      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.3        	2.7.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.2        	2.7.0      	A Helm chart to deploy the New Relic Infrastruc...
[...]
```

Reply:
Should we somehow assert the output of `helm search repo newrelic -l --devel`? For example, if I run it on my machine:

```
helm search repo newrelic -l --devel && echo success
No results found
success
```

For example, this should fail as expected:
```shell
helm search repo newrelic -l --devel | grep -e nri-bundle | grep -- '-pre'
```

Reply:
oh f... I do not know why I was totally sure it was failing if nothing was returned.

I'add a grep